src/toucan/db.clj

```diff
@@ -271,7 +271,11 @@
   "Compile `honeysql-from` and call `jdbc/query` against the application database. Options are passed along to
   `jdbc/query`."
   [honeysql-form & {:as options}]
-  (jdbc/query (connection) (honeysql->sql honeysql-form) options))
+  (jdbc/query (connection)
+              (honeysql->sql honeysql-form)
+              ; FIXME: This has already been fixed in `clojure.java.jdbc`, so
```

Comment:
https://guide.clojure.style/#two-semicolons-for-code-fragment

Reply:
Fixed :wrench: